api/scripts/elastic_ops/.python-version

```diff
@@ -0,0 +1 @@
+elastic_ops
```

Comment:
Shouldn't have committed this, but won't harm anything as this is a dev-only script and a pyenv-only file.